crates/api_models/src/payments.rs

```diff
@@ -276,27 +276,38 @@ pub struct PaymentsRequest {
     /// Passing this object creates a new customer or attaches an existing customer to the payment
     pub customer: Option<CustomerDetails>,
 
-    /// The identifier for the customer object. This field will be deprecated soon, use the customer object instead
-    #[schema(max_length = 255, example = "cus_y3oqhf46pyzuxjbcn2giaqnb44")]
+    /// The identifier for the customer object.
+    /// This field will be deprecated soon, use the customer object instead
+    #[schema(
+        max_length = 255,
+        example = "cus_y3oqhf46pyzuxjbcn2giaqnb44",
+        deprecated
+    )]
+    #[remove_in(PaymentsUpdateRequest, PaymentsCreateRequest, PaymentsConfirmRequest)]
```

Comment:
we should keep this field

Reply:
This is available in `struct CustomerDetails`
Why to keep this here?